projects/aca-shared/rules/src/app.rules.ts

```diff
@@ -624,3 +626,19 @@ export function canOpenWithOffice(context: AcaRuleContext): boolean {
 
   return context.permissions.check(file, ['update']);
 }
+
+function isSmartFolder(context: RuleContext): boolean {
```

Comment:
We should use `AcaRuleContext` otherwise this rule won't be reusable in ADW as far as I remember

Reply:
isSmartFolder has parameter with RuleContext, because canEditAspects and canManagePermissions also have that. In ADW there is imported for example: 
export const canEditAspects = (context: **RuleContext**)
so I think it would be fine if I just leave it like that?

Reply:
Okay fine for me